.github/workflows/trigger-tag.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       # and 2022.1.1-dev9 will become 2022.1.1-dev10
       - name: Get next dev tag name
         run: |
-          LAST_TAG=$(git tag --sort=v:refname --merged ${{ env.BRANCH_NAME }} | grep -e '-dev' | tail -1)
+          LAST_TAG=$(git tag --sort=v:refname --merged ${{ env.BRANCH_NAME }} | grep -E '^[0-9]{4}\.[0-9]+\.[0-9]+-dev[0-9]+$' | tail -1)
```

Comment:
Sanity Check: We are now searching for `####.######-dev######`. We use a `+` for the quarter section and so that would be one and more integer values.

Reply:
Ignore. I got confused.